libraries/chain/controller.cpp

```diff
@@ -4775,9 +4833,21 @@ void controller::assemble_and_complete_block( block_report& br, const signer_cal
    br = my->pending->_block_report;
 }
 
-void controller::commit_block() {
+void controller::commit_block(block_report& br) {
+   fc::time_point start = fc::time_point::now();
+
    validate_db_available_size();
    my->commit_block(block_status::incomplete);
+
```

Comment:
It would be nice those logging are done inside `my-commit_block`.

Reply:
It should only be done for `commit` called by the producer plugin. We could tell that in `my->commit_block()` via `block_status::incomplete`, but I don't see how that is better than logging it here.

Reply:
All other public controller APIs are uniformly one-line code calling the corresponding function in `my`, hiding the implementation. But you don't have to change.

Reply:
Makes sense, I'll move it.